src/redux/categories/categories.js

```diff
@@ -0,0 +1,17 @@
+// Actions
+const STATUS_CHECK = 'STATUS_CHECK';
+
+// Action Creators
+export const statusCheck = () => ({ type: STATUS_CHECK });
+
+// Reducer
+const categories = (state = [], action) => {
```

Comment:
- [ ] Kindly set the `initialState` to an empty array and then assign it to state here as demonstrated earlier.